case_gather/cron_jobs.py

```diff
@@ -6,7 +6,7 @@
 CRONLOGGER = logging.getLogger('cronJobs')
 
 class GatherCases(CronJobBase):
-    RUN_EVERY_MINS = 30
+    RUN_EVERY_MINS = 1
```

Comment:
I think this value seems would benefit by being configurable, especially if it and promises/cronjobs.py should share the same value. Could it be added to a settings file?